Fix for ida 6.2 sdk to compile plugins with gcc 4.8
--- plugin.gcc.mak.orig	2011-09-16 19:04:16 +0000
+++ plugin.gcc.mak	2014-10-12 20:16:53 +0000
@@ -18 +18 @@
-L=$(IDAUNIX)lib/gcc.w32/
+L=$(IDAUNIX)lib/x86_win_gcc_32/
@@ -22 +22 @@
-F=obj/gcc.w32/
+F=obj/x86_win_gcc_32/
@@ -35 +35 @@
-CFLAGS=-I$(I) -DWIN32 -D__NT__ -D__IDP__ -mrtd -mno-cygwin $(__CFLAGS)
+CFLAGS=-I$(I) -DWIN32 -D__NT__ -D__IDP__ -m32 $(__CFLAGS)
@@ -41 +41 @@
-LDFLAGS=--def ../plugin.def -Wl,--dll -shared -mno-cygwin $(_LDFLAGS)
+LDFLAGS=--def ../plugin.def -Wl,--dll -shared -m32 $(_LDFLAGS)
